test/data/basicChartsOptions.js

```diff
@@ -1,5 +1,223 @@
+                emphasis: globalBlurEmphasis,
+            },
+            {
+                name: "Affiliate Ad",
+                type: "line",
+                label: {
+                    show: true,
+                },
+                data: [220, 182, 191, 234, 290, 330, 310],
+                emphasis: globalBlurEmphasis,
+            },
+            {
+                name: "Video Ad",
+                type: "line",
+                label: {
+                    show: true,
+                },
+                data: [150, 212, 201, 154, 190, 330, 410],
+                emphasis: globalBlurEmphasis,
+            },
+        ],
+    },
+    ////////////// Hightlight multilple series(blurScope = 'coordinateSystem') //////////////
+    {
+        id: "highlightMultipleSeries(blurScope='coordinateSystem')",
+        name: "Dispatch action to hightlight multiple series (with blurScope = 'coordinateSystem')",
+        legend: {
+            top: 30,
+            data: [
+                "polar",
+                "Direct",
+                "Mail Ad",
+                "Affiliate Ad",
+                "Video Ad",
+                "Search Engine",
+            ],
+        },
+        xAxis: {
+            type: "category",
+            data: ["Mon", "Tue", "Wed", "Thu", "Fri", "Sat", "Sun"],
+        },
+        yAxis: {
+            type: "value",
+        },
+        polar: {},
+       
+        angleAxis: {
+            type: 'value',
+            startAngle: 0
+        },
+        radiusAxis: {
+        },
+        series: [
+            {
+                coordinateSystem: 'polar',
+                name: 'polar',
+                type: 'line',
+                data: polarData,
+                emphasis: coordinateSystemBlurEmphasis,
+            },
+            {
+                name: "Direct",
+                type: "line",
+                label: {
+                    show: true,
+                },
+                data: [320, 302, 301, 334, 390, 330, 320],
+                emphasis: coordinateSystemBlurEmphasis,
+            },
+            {
+                name: "Mail Ad",
+                type: "line",
+                label: {
+                    show: true,
+                },
+                data: [120, 132, 101, 134, 90, 230, 210],
+                emphasis: coordinateSystemBlurEmphasis,
+            },
+            {
+                name: "Affiliate Ad",
+                type: "line",
+                label: {
+                    show: true,
+                },
+                data: [220, 182, 191, 234, 290, 330, 310],
+                emphasis: coordinateSystemBlurEmphasis,
+            },
+            {
+                name: "Video Ad",
+                type: "line",
+                label: {
+                    show: true,
+                },
+                data: [150, 212, 201, 154, 190, 330, 410],
+                emphasis: coordinateSystemBlurEmphasis,
+            },
+        ],
+    },
```

Comment:
Seems these new cases are mainly target for the hover focus feature. We can append them in `hoverFocus.html` instead of adding them in the `basicChartOptions`. Options in `basicChartOptions` are usually very basic and will be used in tests of many features.

Also, if we want to add a new test case, always adding it at the tail instead of the head. Or it will affect the replay of user interactions in the automatic visual regression testing tool.

Reply:
got it, fixed~